### PR TITLE
test/upgrade: Added fast local test to upgrade scenario

### PR DESCRIPTION
**Pull request by Ebaneck**
**Component**:

'upgrade', 'test'

**Context**: 

See #1699 

**Summary**:

We need to be able to verify that a Metalk8s platform is fully functional before and after an upgrade.

**Acceptance criteria**: 

- Upgrade local test should pass.
- Successful upgrades should be tested.
- CI green build
---

Closes: #1699

### CODE REVIEW

File: eve/main.yml [hunk 1]
```diff
@@ -360,6 +360,17 @@ stages:
             /srv/scality/metalk8s-%(prop:metalk8s_prev_version)s/bootstrap.sh
             --verbose
           haltOnFailure: true
+      - ShellCommand: *install_kubectl
```
Comment (TeddyAndrieux) [inline eve/main.yml]: It's not needed. Install of kubectl is part of the bootstrap "now". Btw this should be removed from every step

File: eve/main.yml [hunk 2]
```diff
@@ -370,6 +376,16 @@ stages:
             /srv/scality/metalk8s-%(prop:metalk8s_version)s/upgrade.sh
             --destination-version %(prop:metalk8s_version)s
           haltOnFailure: true
+      - ShellCommand:
+          name: Run fast tests locally for %(prop:metalk8s_version)s
+          env:
+            ISO_MOUNTPOINT: >
+              /srv/scality/metalk8s-%(prop:metalk8s_version)s
+          command: >
+            git checkout -b %(prop:branch)s &&
```
Comment (sayf-eddine-scality) [inline eve/main.yml]: git checkout without -b

**Review by TeddyAndrieux — DISMISSED** [review]
LGTM

**Review by sayf-eddine-scality — DISMISSED** [review]

**Review by sayf-eddine-scality — APPROVED** [review]

**Pull request merged**